src/jdk.incubator.jextract/share/classes/jdk/internal/clang/libclang/constants$0.java

```diff
@@ -32,6 +32,8 @@
 import java.nio.ByteOrder;
 import jdk.incubator.foreign.*;
 import static jdk.incubator.foreign.CLinker.*;
+import static jdk.internal.clang.libclang.CLayouts.*;
+
```

Comment:
Same comment as for RuntimeHelper. All these are extracted code. Need to figure out how to do CLayouts equivalent from jextract.

Reply:
This is just a workaround I did to avoid rewriting all these constants classes (since they were using layouts like C_INT and such). So I defined the `CLayouts` class, and kept the rest identical to what it was before. If we regenerate all the bindings, all these dependencies will go away.